examples/oft-aptos/evmOAppConfig.ts

```diff
@@ -0,0 +1,121 @@
+import { ExecutorOptionType } from '@layerzerolabs/lz-v2-utilities-v3'
+
+import type { OAppNodeConfig, OAppEdgeConfig } from '@layerzerolabs/toolbox-hardhat'
+
+const bscAccountConfig: OAppNodeConfig = {
+    owner: '0xCD5691EA878D262516be278cb46f2fd7fF132083',
+    delegate: '0xCD5691EA878D262516be278cb46f2fd7fF132083',
+}
+
+const ethAccountConfig: OAppNodeConfig = {
+    owner: '0xCD5691EA878D262516be278cb46f2fd7fF132083',
+    delegate: '0xCD5691EA878D262516be278cb46f2fd7fF132083',
+}
+
+const bscOAppConfig: OAppEdgeConfig = {
+    enforcedOptions: [
+        {
+            msgType: 1,
+            optionType: ExecutorOptionType.LZ_RECEIVE,
+            gas: 200000, // gas limit in wei for EndpointV2.lzReceive
+            value: 0, // msg.value in wei for EndpointV2.lzReceive
+        },
+        {
+            msgType: 2,
+            optionType: ExecutorOptionType.LZ_RECEIVE,
+            gas: 200000, // gas limit in wei for EndpointV2.lzCompose
+            value: 0, // msg.value in wei for EndpointV2.lzCompose
+        },
+        {
+            msgType: 1,
+            optionType: ExecutorOptionType.NATIVE_DROP,
+            amount: 0, // amount of native gas token in wei to drop to receiver address
+            receiver: '0x0000000000000000000000000000000000000000',
+        },
+    ],
+    sendLibrary: '0xFaa067f96816bc1045986a742f914179fA5924f8',
```

Comment:
If this is a BSC Testnet configuration it seems wrong, eg. sendLibrary should be: `0x55f16c442907e86D764AFdc2a07C2de3BdAc8BB7`.

Check here: https://docs.layerzero.network/v2/developers/evm/technical-reference/deployed-contracts#bsc-testnet

Reply:
EVM testing is not testnet compliant. That is for a local test on sandbox. It is work in progress